polkadot-parachains/statemine/src/weights/pallet_utility.rs

```diff
@@ -29,16 +46,16 @@ use sp_std::marker::PhantomData;
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_utility::WeightInfo for WeightInfo<T> {
 	fn batch(c: u32, ) -> Weight {
-		(16_177_000 as Weight)
-			// Standard Error: 0
-			.saturating_add((4_582_000 as Weight).saturating_mul(c as Weight))
+		(31_019_000 as Weight)
+			// Standard Error: 3_000
+			.saturating_add((7_923_000 as Weight).saturating_mul(c as Weight))
```

Comment:
this seems quite a bit heavier

Reply:
might be the events from here? https://github.com/paritytech/substrate/pull/9462